emission/simulation/test_prob140.py

```diff
@@ -0,0 +1,30 @@
+import unittest
```

Comment:
again, unittests go into `emission/tests`

Reply:
@alvinalexander please move unittests to `emission/tests` and ensure that they pass with `runAllTests.sh`